### PR TITLE
[DOCS] HTTP client stats

### PR DESCRIPTION
**Pull request by danhermann**
Adds documentation for the stats added in #64561.

Preview: https://elasticsearch_70512.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/cluster-nodes-stats.html

### CODE REVIEW

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 1]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
++
+.Properties of `clients`
+[%collapsible%open]
+=======
+`id`::
+(integer)
+Unique ID for the HTTP client.
+`agent`::
+(string)
+Reported agent for the HTTP client, if available.
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1895]:
```suggestion
Reported agent for the HTTP client. If unavailable, this property is not
included in the response.
```

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 2]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
+`local_address`::
+(string)
+Local address for the HTTP client.
+`remote_address`::
+(string)
+Remote address for the HTTP client.
+`forwarded_for`::
+(string)
+Value from the client's `x-forwarded-for` HTTP header, if available.
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1904]:
```suggestion
Value from the client's `x-forwarded-for` HTTP header. If unavailable, this
property is not included in the response.
```

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 3]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
+`opqaue_id`::
+(string)
+Value from the client's `x-opaque-id` HTTP header, if available.
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1907]:
```suggestion
Value from the client's `x-opaque-id` HTTP header. If unavailable, this property
is not included in the response.
```

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 4]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
+Time at which the client closed the connection if the connection is closed.
+`last_request_time_millis`::
+(integer)
+Time of the most recent request from this client.
+`request_count`::
+(integer)
+Number of requests from this client.
+`request_size_bytes`::
+(integer)
+Cumulative size in bytes of all requests from this client
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1922]:
```suggestion
Cumulative size in bytes of all requests from this client.
```

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 5]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
+`local_address`::
+(string)
+Local address for the HTTP client.
+`remote_address`::
+(string)
+Remote address for the HTTP client.
+`forwarded_for`::
+(string)
+Value from the client's `x-forwarded-for` HTTP header, if available.
+`opqaue_id`::
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1905]: Should this be `x_opqaue_id` per https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/http/HttpStats.java#L81?
```suggestion
`x_opqaue_id`::
```

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 6]
```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
...
+`agent`::
+(string)
+Reported agent for the HTTP client, if available.
+`local_address`::
+(string)
+Local address for the HTTP client.
+`remote_address`::
+(string)
+Remote address for the HTTP client.
+`forwarded_for`::
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1902]: Should this be `x_forwarded_for` per https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/http/HttpStats.java#L80?
```suggestion
`x_forwarded_for`::
```

Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1902]: It looks like the `last_uri` property may be missing:
https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/http/HttpStats.java#L74

**Review by jrodewig — COMMENTED** [review]
Thanks @danhermann. I left a few suggestions and questions.

In some cases, it wasn't clear if a property would be returned if it was unavailable. I also didn't know if a few fields were misnamed/missing. I'll take another look afterward. Thanks again!

File: docs/reference/cluster/nodes-stats.asciidoc (hunk 6 above)
Comment (danhermann) [reply docs/reference/cluster/nodes-stats.asciidoc]: Thanks, @jrodewig! I missed a bunch of stuff on this one. I add the missing `last_uri` stat in https://github.com/elastic/elasticsearch/pull/70512/commits/aa3efd4d60893711575fa91633ae6915bf580b41.

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 7]
```diff
@@ -1879,6 +1879,54 @@ Current number of open HTTP connections for the node.
...
+(array of objects)
+Information on current and recently-closed HTTP client connections.
++
+.Properties of `clients`
+[%collapsible%open]
+=======
+`id`::
+(integer)
+Unique ID for the HTTP client.
+`agent`::
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-stats.asciidoc:1893]: Nit: I'd add a newline between each definition for readability. Won't affect the HTML output tho.

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks @danhermann!

**Pull request merged**